projects/addon-table/components/table/directives/cell.directive.ts

```diff
@@ -1,6 +1,12 @@
 import {Directive, Inject, Input, TemplateRef} from '@angular/core';
 import {tuiDefaultProp} from '@taiga-ui/cdk';
 
+/**
+ * @deprecated
+ * It is required only by {@link TuiTrComponent}. Can be dropped after its deletion.
+ * ___
+ * TODO v4.0 delete it.
+ */
 @Directive({
```

Comment:
tuiCell is necessary in order to rearrange columns by `columns` array of `tuiTable` directive.

Reply:
<img width="1227" alt="Screenshot 2022-10-12 at 14 39 17" src="https://user-images.githubusercontent.com/35179038/195333349-4939f5ab-02d7-4fab-a237-57b32cd6017c.png">

Reply:
fixed